### DISCUSSION TITLE
Download Issues

### DISCUSSION THREAD

**Discussion by jsfl823** [JunkFood02/Seal#1649] [General]
Download error always occurs How to solve this problem

**Comment by darlopvil** [comment]
me 2. Even with tiktok that never gave issues. I'm using the latest preview build:

 App version: 2.0.0-alpha.2 (20002)
Device information: Android 14 (API 34)
Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]
Yt-dlp version: 2024.08.19.232821

Error message: 
App version: 2.0.0-alpha.2 (20002)
Device information: Android 14 (API 34)
Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]
Yt-dlp version: 2024.08.19.232821

URL: https://vm.tiktok.com/ZGe3y7JQQ/
ERROR: [TikTok] 7281040383713266976: Video not available, status code 10222; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

**Comment by ssupshub** [comment]
How can I download seal in pc ?

**Comment by iamFERU** [comment]
App version: 1.13.1 (11312)
Device information: Android 15 (API 35)
Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]
Yt-dlp version: 2024.11.09.232836

URL: https://x.com/HasnaZaruriHai/status/1855249419601342507?s=09
ERROR: Unable to download video: [Errno 36] File name too long: '/storage/emulated/0/Download/Seal/tmp/HasnaZarooriHai🇮🇳 - मैंने तो सुना था कि जया किशोरी जी कथा सुनाती हैं,  मगर ये क्या है？ 😧😧 [1855249276567228416].fhls-audio-128000-Audio.mp4.ytdl'


Facing this error. 
Any fix for this?
